service/src/main/java/org/dice_research/fc/sparql/query/QueryEngineCustomHTTP.java

```diff
@@ -37,6 +37,8 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(QueryEngineCustomHTTP.class);
 
+    int tryNumber;
```

Comment:
I don't think that this should be an attribute. It is only used in the `createRequest` method and it creates problems if COPAAL tries to send several queries at once.

I would suggest a local variable:
```java
    public ResultSet execSelect() {
      String result = createRequest();
      ...
    }

    private String createRequest() {
      return createRequest(0);
    }

    private String createRequest(int tryNumber) {
      ...
      if(result.contains("404 File not found") && tryNumber < 5){
        ...
        createRequest(tryNumber + 1);
      }
      ...
    }
```